platform/src/test/java/org/exoplatform/platform/qa/ui/platform/calendar/CalendarPublishActivityTestIT.java

```diff
@@ -0,0 +1,791 @@
+    calendarManagement.saveAddCalendar();
+    $(byXpath(ELEMENT_CALENDAR_LIST_ITEM.replace("$calendar", shareCalendar))).should(Condition.visible);
+    calendarManagement.shareCalendar(shareCalendar, groupShare, edit, 1);
+
+    homePagePlatform.goToCalendarPage();
+    info("create event for share calendar");
+    calendarManagement.executeActionCalendar(shareCalendar, CalendarManagement.menuOfCalendarOption.ADDEVENT);
+    eventManagement.inputDataEventInQuickForm(titleEventShare,
+                                              titleEventShare,
+                                              getDate(1, "MM/dd/yyyy HH") + ":00",
+                                              getDate(1, "MM/dd/yyyy HH") + ":30",
+                                              false);
+    eventManagement.saveQuickAddEvent();
+
+    homePagePlatform.goToHomePage();
+
+    info("Test 3: Activity isn't publish for a event of Personal, Shared and other group calendars than Space");
+    info("Event/task of personal/group/share calendar are NOT displayed on homepage activity");
+    $(byXpath(ELEMENT_ACTIVITY_TASK_EVENT_TITLE.replace("$name", titleEventShare))).shouldNot(Condition.visible);
+
+    homePagePlatform.goToCalendarPage();
+    calendarManagement.deleteCalendar(shareCalendar, false);
+
+  }
+
+  @Test
+  public void test03_VerifyActivityEventOfGroupCalendars() {
+    String titleEventGroup = "titleEventGroup" + getRandomNumber();
+    String groupCalendar = "Content Management";
+    homePagePlatform.goToCalendarPage();
+    info("create event for group calendar");
+    calendarManagement.executeActionCalendar(groupCalendar, CalendarManagement.menuOfCalendarOption.ADDEVENT);
+    eventManagement.inputDataEventInQuickForm(titleEventGroup,
+                                              titleEventGroup,
+                                              getDate(1, "MM/dd/yyyy HH") + ":00",
+                                              getDate(1, "MM/dd/yyyy HH") + ":30",
+                                              false);
+    eventManagement.saveQuickAddEvent();
+    homePagePlatform.goToHomePage();
+    info("Test 3: Activity isn't publish for a event of Personal, Shared and other group calendars than Space");
+    info("Event/task of personal/group/share calendar are NOT displayed on homepage activity");
+    $(byXpath(ELEMENT_ACTIVITY_TASK_EVENT_TITLE.replace("$name", titleEventGroup))).shouldNot(Condition.visible);
+
+    info("Clear data");
+    homePagePlatform.goToCalendarPage();
+    calendarHomePage.deleteEventTask(titleEventGroup,
+                                     CalendarHomePage.selectViewOption.MONTH,
+                                     CalendarHomePage.selectDayOption.DETAILTIME,
+                                     getDate(1, "MMM dd yyyy"),
+                                     false,
+                                     false);
+  }
+
+  @Test
+  @BugInPLF("CAL-1296")
```

Comment:
This issue has been fixed. To be removed.

Reply:
done